### PR TITLE
Added missing datepicker options pickDate.

### PR DESCRIPTION
**Pull request by Fuck4ik**
Added the ability for datetime to display only time. The default icon now changes depending on the pickDate option

## Changelog

```markdown
### Added
- Added dp_option "dp_pick_date" in BasePickerType

### Changed
- Changed template datepicker.html.twig to display the time icon

### CODE REVIEW

**Comment by greg0ire** [comment]
You could have just force pushed instead of creating a new PR...

**Comment by Fuck4ik** [comment]
Thanks, in the future I'll know it

File: Form/Type/BasePickerType.php [hunk 1]
```diff
@@ -125,6 +125,7 @@ protected function getCommonDefaults()
             'widget' => 'single_text',
             'datepicker_use_button' => true,
             'dp_pick_time' => true,
+            'dp_pick_date' => true,
```
Comment (greg0ire) [inline Form/Type/BasePickerType.php]: @jlamur not sure there is a need for the expression you proposed if there is this default value.

Comment (Fuck4ik) [reply Form/Type/BasePickerType.php]: pickDate = true, this is standard behavior sonata_type_datetime_picker. I just expanded the option to turn off date selection.

Comment (jlamur) [reply Form/Type/BasePickerType.php]: I though about this hypothetical guy who extends this class, overrides the `getCommonDefaults()` method and doesn't call `parent::getCommonDefaults()` :thinking:

We will let this guy deal with this on its own. :)

Comment (greg0ire) [reply Form/Type/BasePickerType.php]: Good thinking, but that's a bit like spacebar heating :P

**Comment by greg0ire** [comment]
@sonata-project/contributors please review

**Comment by greg0ire** [comment]
Thanks @Fuck4ik

**Pull request merged**